zerver/openapi/zulip.yaml

```diff
@@ -12750,6 +12750,57 @@ paths:
                     description: |
                       An example JSON error response for when the public data export
                       exceeds the maximum allowed data export size.
+  /export/realm/consents:
+    get:
+      operationId: get-realm-export-consents
+      summary: Get all private data export consents
+      tags: ["server_and_organizations"]
+      x-requires-administrator: true
+      description: |
+        Fetch the consents of users for their [private data exports](/help/export-your-organization#full-export-with-member-consent)
+        of the organization.
+
+        **Changes**: New in Zulip 10.0 (feature level 294).
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      export_consents:
+                        type: array
+                        description: |
+                          An array of objects where each object contains a user_id
+                          and the user's consent for private data export.
```

Comment:
"contains a user ID and whether the user has consented for their private data to be exported"